### ISSUE TITLE
Feature and feature value selects are missing search

### ISSUE THREAD

**Issue by Hlavtox** [PrestaShop/PrestaShop#34607]
### Describe the bug and add attachments

Feature and feature value inputs on new product page are missing a search feature. You must scroll the list manually.

It's added only for new features, if you click `Add feature`. When you save the page, the search gets lost again.

![Snímek obrazovky 2023-11-17 141306](https://github.com/PrestaShop/PrestaShop/assets/6097524/03384f6b-c35c-436b-8e03-f5ccd2fdb72d)

### Steps to reproduce

1. Enable new product page.
2. Make a feature with like 15 values.
3. Go add it to a product, see that the search is missing.

### PrestaShop version(s) where the bug happened

8.1.2

**Comment by M0rgan01** [comment]
I confirm that search is no longer present in product v2

Product v1
![image](https://github.com/PrestaShop/PrestaShop/assets/36233446/b8f9f865-8eaa-4aab-9694-5ccf09aeabfe)
Product v2
![image](https://github.com/PrestaShop/PrestaShop/assets/36233446/cf0cebc7-88c6-4eb6-96ff-cec0d16ceb16)

EDIT

Search only appears if we add a feature

![image](https://github.com/PrestaShop/PrestaShop/assets/36233446/aa98742c-b77e-4bc5-a821-40f61b54a20e)

**Comment by AureRita** [comment]
Hi @Hlavtox 

Thank you for your report, I tested it on 8.1.3 and 8.1.1, but currently I can't reproduce your issue as you can see : 

[Untitled_ Nov 17, 2023 4_26 PM.webm](https://github.com/PrestaShop/PrestaShop/assets/100699445/f946df81-9cc5-4cb9-9fb1-240aac85bdda)

@PrestaShop/quality-council Someone could reproduce the issue ? 

Waiting for feedback

**Comment by Hlavtox** [comment]
@AureRita You must save the product, you will see that it's missing

**Comment by AureRita** [comment]
@Hlavtox 

Indeed, I reproduce it with the last provided information, thank you

**Comment by Hlavtox** [comment]
Fixed by https://github.com/PrestaShop/PrestaShop/pull/36244
